modules/ROOT/pages/7.3-release-notes.adoc

```diff
@@ -141,6 +141,35 @@ As a result, non-breaking spaces between different inline elements are now norma
 
 {productname} {release-version} also includes the following bug fix<es>:
 
+=== Sequential html comments would in some cases generate unwanted elements.
+// #TINY-10955
+
+Previously in {productname}, sequential HTML comments would generate unwanted elements in the editor. For example, inserting the following HTML into the editor would result in an `+&nbsp;+` element being inserted:
+
+.Unwanted elements generated in the editor
+[source, html]
+----
+<div>a</div>
+<!-- comment -->
+<p>&nbsp;</p>
+<!-- comment -->
+<div>b</div>
+----
+
+As a consequence, whitespace was added between the comments, resulting in extra paragraphs being generated in the editor.
+
+In {productname} {release-version}, this issue has been resolved. The editor now correctly handles sequential HTML comments by removing the whitespace before parsing.
+
+.Resolved sequential HTML comments whitespace
+[source, html]
+----
+<div>a</div>
+<!-- comment --><!-- comment -->
+<div>b</div>
+----
+
+As a result, no extra paragraphs are generated.
```

Comment:
I don't think the current version accurately explains the behaviour.
From reading the ticket, it appears the issue was that we start with
```
<div>a</div>
<!-- comment -->
<!-- comment -->
<div>b</div>
```
and the editor would add the extra
```
<p>&nbsp;</p>
```
Based on that, maybe something like the below?
Diff: https://www.diffchecker.com/NDcybMAt/
```suggestion
=== Sequential html comments would in some cases generate unwanted elements.
// #TINY-10955

Previously in {productname}, sequential HTML comments would generate unwanted elements in the editor. For example, inserting the following HTML into the editor would result in an `+&nbsp;+` element being inserted:

.Unwanted elements generated in the editor
[source, html]
----
<div>a</div>
<!-- comment -->
<p>&nbsp;</p>
<!-- comment -->
<div>b</div>
----

As a consequence, whitespace was added between the comments, resulting in extra paragraphs being generated in the editor.

In {productname} {release-version}, this issue has been resolved. The editor now correctly handles sequential HTML comments by removing the whitespace before parsing.

.Resolved sequential HTML comments whitespace
[source, html]
----
<div>a</div>
<!-- comment --><!-- comment -->
<div>b</div>
----

As a result, no extra paragraphs are generated.
```

Reply:
Not a big deal if we wanna keep the original title, but I believe `consecutive` is the correct term here, instead of `sequential`.

Reply:
Going by the diffchecker, yeah, seems about right.